pyvista/core/filters/poly_data.py

```diff
@@ -1080,7 +1085,10 @@ def tube(
             field = poly_data.get_array_association(scalars, preference=preference)
             # args: (idx, port, connection, field, name)
             tube.SetInputArrayToProcess(0, 0, 0, field.value, scalars)
-            tube.SetVaryRadiusToVaryRadiusByScalar()
+            if not absolute:
+                tube.SetVaryRadiusToVaryRadiusByScalar()
+            else:
+                tube.SetVaryRadiusToVaryRadiusByAbsoluteScalar()
```

Comment:
It is better not to use "not" in if sentence.
```suggestion
            if absolute:
                tube.SetVaryRadiusToVaryRadiusByAbsoluteScalar()
            else:
                tube.SetVaryRadiusToVaryRadiusByScalar()
```